### PR TITLE
Add scripts for provisioning a sandbox

### PR DESCRIPTION
**Pull request by andy-armstrong**

### CODE REVIEW

**Comment by edx-webhook** [comment]
## PR Cover Letter
- [ ] A detailed description of the changes in the body of the PR
- [ ] If available, a link to the relevant JIRA ticket in the body of the PR
  - `[FEDX-XXXX](https://openedx.atlassian.net/browse/FEDX-XXXX)`

## Testing
- [ ] i18n
- [ ] Responsive design
- [ ] RTL

## Reviewers
If you've been tagged for review, please check your corresponding box once you've given the :+1:.
- [ ] Code review: @Reviewer1
- [ ] Code review: @Reviewer2
- [ ] Documentation review: @...
- [ ] UX review: @...
- [ ] Accessibility review: @...
- [ ] Product review: @...

FYI: Tag anyone who might be interested in this PR here.

## Post-review
- [ ] Squash commits
- [ ] Publish packages/doc site

**Comment by andy-armstrong** [comment]
@feanil @benpatterson I have written scripts that provision a sandbox with the Pattern Library Jekyll documentation site (what will become http://ux.edx.org/) so that folks internally can preview the changes. The piece I'm missing is how to expose the port externally. I've been trying to add an nginx configuration based upon the CMS one, but I'm clearly doing something wrong, as this URL doesn't work:

https://pattern-library-andy-armstrong.sandbox.edx.org/index.html

What should I be doing here? I'm sure there's an easier approach that I'm missing... Thanks!

File: scripts/pattern-library.nginx [hunk 1]
```diff
@@ -0,0 +1,57 @@
+server {
+  listen 80;
+  listen 443 ssl;
+
+  server_name pattern-library.*;
```
Comment (feanil) [inline scripts/pattern-library.nginx]: remove the `.`

**Comment by benpatterson** [comment]
Hmm should the provisioning be built via ansible? I could see re-using the rbenv role, for example.

**Comment by andy-armstrong** [comment]
Thanks for the comments. I spoke with devops today and @fredsmith is trying a different approach where I can publish a Jekyll site to an S3 bucket. Given that, I'll close this PR for now.

**Pull request closed without merging**